### DISCUSSION TITLE
QQ交流群

### DISCUSSION THREAD

**Discussion by yang1206** [nutui-uniapp/nutui-uniapp#242] [Announcements]
群号：`873252418`

@RJQingHuan @xiaohe0601 @Yer11214 有什么问题大家可以在 qq 群里沟通

**Comment by RJQingHuan** [comment]
qq群？搜不到啊在 2024年3月8日，20:00，yang1206 ***@***.***> 写道：﻿
群号：873252418
@RJQingHuan @xiaohe0601 @Yer11214

**Comment by yang1206** [comment]
现在可以正常搜索到了，刚刚默认是私密的
